python/paddle/fluid/tests/unittests/test_fuse_elewise_add_act_pass.py

```diff
@@ -87,8 +89,72 @@ def test_batchnorm_fc_with_fuse_op(self):
         )
 
 
-if __name__ == '__main__':
-    import paddle
+class TestFuseActElewiseAddInplaceGradPass(unittest.TestCase):
+    def build_program(self, main_program, startup_program):
+        with fluid.program_guard(main_program, startup_program):
```

Comment:
try to use paddle.static.program_guard instead of `fluid api`

Reply:
Done, thanks!